docs/4.Integrations/pre-commit.md

```diff
@@ -7,7 +7,7 @@ nav_order: 6
 
 # Pre-Commit
 
-If you want to automatically run `checkov` when files in your git repo change, [install the pre-commit binary](https://pre-commit.com/#install), and add a [.pre-commit-config.yaml file](./.pre-commit-config.yaml) to your project with content similar to the example below.
+If you want to automatically run `checkov` when files in your git repo change, [install the pre-commit binary](https://pre-commit.com/#install), and add a [.pre-commit-config.yaml file](https://github.com/bridgecrewio/checkov/blob/main/.pre-commit-config.yaml) to your project with content similar to the example below.
```

Comment:
To automatically run `Checkov` whenever files in your Git repository change, first [install the pre-commit binary](https://pre-commit.com/#install) and then add a [.pre-commit-config.yaml] file (https://github.com/bridgecrewio/checkov/blob/main/.pre-commit-config.yaml)) to your project with content similar to the example below.